app/file.go

```diff
@@ -684,7 +684,11 @@ func (t *UploadFileTask) init(a *App) {
 	}
 	t.teeInput = io.TeeReader(t.limitedInput, t.buf)
 
-	t.pluginsEnvironment = a.GetPluginsEnvironment()
+	plgEnvironment, appErr := a.GetPluginsEnvironment()
+	if appErr != nil {
+		mlog.Error("Plugin is disabled or not set", mlog.Err(appErr))
```

Comment:
Gentle reminder to check our logging [principles](https://developers.mattermost.com/contribute/server/style-guide/#log-levels) before merging this change.

Reply:
/check-cla